scripts/backend-config-migration.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# This script will migrate the current backend config to the latest template.
+# A backup file is created. Existing backups will be overwritten.
+#
+# How to use
+#
+# ./backend-config-migration.sh <instance>
+# example: ./backend-config-migration.sh qm
+#
+# Attention: on macos, see setEnv function and enable the macos line instead the linux line
+#
+
+##############################
+# setup
+##############################
+
+source "../setup.env"
+
+##############################
+# ask for input data
+##############################
+
+if [ -n "$1" ]; then
+  instance="$1"
+else
+  echo "What is the name of the instance?"
+  read -r instance
+fi
+
+##############################
+# variables
+##############################
+
+path="../../$PREFIX$instance"
+isBackendEnabled=0
+
+# setting backend version. Pinned to prevent config conflicts
+backendVersion=v1.16.0
```

Comment:
maybe move to a central ENV so that this can update with our version update scripts without constant updates to git?

Reply:
optional, otherwise find latest version from git like we do for frontend?
https://github.com/Aam-Digital/ndb-setup/blob/f6e39915fe477b65532daac53de485d182adb606/interactive_setup.sh#L104C1-L106

Reply:
@tomwwinter , does the migration still explicitly depend on a version or should we also change this to pull the latest release by default, like in the enable-backend script?